docs/2.0/docs/accountfactory/architecture/security-controls.md

```diff
@@ -0,0 +1,231 @@
+# Controls
+
+Gruntwork Account Factory employs a defense-in-depth approach to secure workflows across both GitHub and GitLab platforms. This document outlines the controls Pipelines uses to ensure that only infrastructure written in code and approved by a reviewer can be deployed to your AWS accounts.
+
+Account Factory relies on Pipelines to drive infrastructure changes via GitOps workflows, so make sure to read the [Pipelines security controls](/2.0/docs/pipelines/architecture/security-controls) for more details on how Pipelines secures workflows.
+
+## Least privilege principle
+
+Pipelines adheres to the principle of least privilege, granting only the necessary permissions for infrastructure actions.
+
+By default, the only repository/group required to interact with infrastructure using Pipelines in Gruntwork Account Factory is the `infrastructure-live-root` repository/group. This contains infrastructure code for `management`, `logs`, `security`, and `shared` accounts. Access should be limited to a small, trusted group responsible for defining critical infrastructure, similar to the role of the `root` user in Unix systems.
```

Comment:
I'm a little confused by this section.

Does Pipelines itself adhere to the principle of least privilege...by having separate roles for plan/apply?

Or is the idea that the infra-live-root repo is the only one with OIDC access to the management, logs, security, and shared accounts? If so, that doesn't seem like a Pipelines feature as much as a configuration choice.